documentation/index.md

```diff
@@ -133,13 +147,14 @@ execution_time = time() - t0
 assert execution_time < 1.5  # a() and b() are executed in parallel
 print(f"Graph execution took {execution_time:.2f} seconds")
 print(f"res = {res}")
-
+## Graph execution took 1.00 seconds
+## 'A + B = C'
 ```
-As you can see, the execution time of pipeline takes less than 2 seconds, which means that some part of the code ran in parallel to the other
+**As you can see, the execution time of pipeline takes less than 2 seconds, which means that some part of the code ran in parallel to the other**
 
-### `DAG` is like a normal function
+### **Mastering `DAG` usage**
```

Comment:
I don't think we should have a paragraph whose title is Mastering DAG usage and then afterwards another one named advanced dag usage...